frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -193,85 +213,104 @@ const CollectiveOrganizing = () => (
  */
 export const FaqsContent: Faq[] = [
   {
-    question: "I'm scared. What happens if my landlord retaliates?",
+    question: li18n._(t`I'm scared. What happens if my landlord retaliates?`),
     category: "After Sending Your Letter",
     previewOptions: {
       priorityInPreview: 1,
       answerPreview: (
+        <Trans>
+          <p>
+            It’s normal to feel anxious or scared that your landlord will
+            retaliate. If your landlord is harassing you, denying you repairs,
+            or trying to illegally evict you, reach out to legal assistance at{" "}
+            <LawHelpLink /> and connect with tenant organizers at{" "}
+            <RightToTheCityLink />.
+          </p>
+        </Trans>
+      ),
+    },
+    answerFull: (
+      <Trans>
         <p>
           It’s normal to feel anxious or scared that your landlord will
           retaliate. If your landlord is harassing you, denying you repairs, or
           trying to illegally evict you, reach out to legal assistance at{" "}
           <LawHelpLink /> and connect with tenant organizers at{" "}
           <RightToTheCityLink />.
         </p>
-      ),
-    },
-    answerFull: (
-      <p>
-        It’s normal to feel anxious or scared that your landlord will retaliate.
-        If your landlord is harassing you, denying you repairs, or trying to
-        illegally evict you, reach out to legal assistance at <LawHelpLink />{" "}
-        and connect with tenant organizers at <RightToTheCityLink />
-        .s
-      </p>
+      </Trans>
     ),
   },
   {
-    question: "Is this free?",
+    question: li18n._(t`Is this free?`),
     category: "Letter Builder",
     previewOptions: {
       priorityInPreview: 2,
       answerPreview: (
+        <Trans>
+          <p>
+            Yes, this is a free website created by 501(c)3 non-profit
+            organizations across the United States.
+          </p>
+        </Trans>
+      ),
+    },
+    answerFull: (
+      <Trans>
         <p>
           Yes, this is a free website created by 501(c)3 non-profit
           organizations across the United States.
         </p>
-      ),
-    },
-    answerFull: (
-      <p>
-        Yes, this is a free website created by 501(c)3 non-profit organizations
-        across the United States.
-      </p>
+      </Trans>
     ),
   },
   {
-    question: "Do I have to go to the post office to mail  my letter?",
+    question: li18n._(
+      t`Do I have to go to the post office to mail  my letter?`
```

Comment:
Should there be two spaces between "mail" and "my"?

Reply:
nope! typo